scale/job/models.py

```diff
@@ -217,9 +217,50 @@ def get_details(self, job_id):
             'superseded_by_job', 'superseded_by_job__job_type'
         ).get(pk=job_id)
 
+        # Attempt to get related job executions
+        if job.status in ['RUNNING', 'COMPLETED', 'FAILED', 'QUEUED']:
+            job_exe = JobExecution.objects.filter(job=job, exe_num=job.num_exes).select_related('job', 'node', 'error')
+            job.execution = job_exe.defer('job__input', 'job__output').order_by('-created')
+        else:
+            job.execution = []
```

Comment:
We should reuse the `JobExecutionManager.get_job_exe_details()` method here.
Also update that method to select related on `jobexecutionend__error` and defer `job__input` and `job__output`.

We should use a try/except block on `JobExecution.DoesNotExist` (just like the execution detail view) and set `job.execution = None` if there is no execution.